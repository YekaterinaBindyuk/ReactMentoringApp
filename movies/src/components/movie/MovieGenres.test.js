import React from "react";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const genres = ["Comedy", "Crime"].join(", ");
  const tree = renderer.create(<div className="genres">{genres}</div>).toJSON();
  expect(tree).toMatchSnapshot(genres);
});
