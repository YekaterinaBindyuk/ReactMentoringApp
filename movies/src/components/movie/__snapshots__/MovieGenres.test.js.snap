// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly: Comedy, Crime 1`] = `
<div
  className="genres"
>
  Comedy, Crime
</div>
`;
